modules/xfeatures2d/include/opencv2/xfeatures2d.hpp

```diff
@@ -179,6 +179,55 @@ class CV_EXPORTS_W LATCH : public Feature2D
     CV_WRAP static Ptr<LATCH> create(int bytes = 32, bool rotationInvariance = true, int half_ssd_size = 3, double sigma = 2.0);
 };
 
+/*
+* BEBLID Descriptor
+*/
+
+/** @brief Class implementing BEBLID (Boosted Efficient Binary Local Image Descriptor)
+ * @cite Suarez2020BEBLID .
```

Comment:
`@cite` is merged to `@brief` without an empty line.

Reply:
I fixed it with the phrase ", described in [XXX]" as other descriptors do.